src/bundle/Resources/public/scss/_icons.scss

```diff
@@ -88,10 +88,10 @@
     }
 }
 
-.ez-field-edit--ezdate,
-.ez-field-edit--ezdatetime,
-.ez-field-edit--eztime {
-    .ez-data-source__btn--clear-input {
+.ibexa-field-edit--ezdate,
+.ibexa-field-edit--ezdatetime,
+.ibexa-field-edit--eztime {
+    .ibexa-data-source__btn--clear-input {
```

Comment:
Do we still need this, since we have clear button by default in `ibexa-input--text`?